WordPress/src/main/java/org/wordpress/android/viewmodel/giphy/provider/TenorProvider.kt

```diff
@@ -99,8 +139,8 @@ internal class TenorProvider @JvmOverloads constructor(
      */
     private inline fun handleResponse(
         response: GifsResponse,
-        onSuccess: (List<GiphyMediaViewModel>, Int?) -> Unit
-    ) {
+        crossinline onSuccess: (List<GiphyMediaViewModel>, Int?) -> Unit
```

Comment:
Yeah, I got way too exaggerated with this method extraction, I've inlined the operation directly into the onSuccess and onFailure declaration since they're simple enough to do so, it got way easier to understand now. I've fixed this point in baade561fe17ed132c6b959a64230aaa75055524.